larrecodnn/ImagePatternAlgs/Tensorflow/Modules/EvaluateROIEff_module.cc

```diff
@@ -2,9 +2,15 @@
 // Class:       EvaluateROIEff
 // Plugin Type: analyzer (art v3_05_00)
 // File:        EvaluateROIEff_module.cc
-//
+// 
```

Comment:
This PR includes a lot of whitespace-only changes.  My guess is you worked on this PR before updating to LArSoft v9?   If so,  please try to rebase this code on top of v9 so that you can avoid these formatting issues.